### PR TITLE
python-click: update to 7.0.

### PR DESCRIPTION
**Pull request by xaltsc**

### CODE REVIEW

File: srcpkgs/python-click/template [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-wrksrc="click-${version}"
+wrksrc="Click-${version}"
 build_style="python-module"
+pycompile_module="click"
 hostmakedepends="python-setuptools python3-setuptools"
 depends="python"
-pycompile_module="click"
 short_desc="Python2 package for creating beautiful command line interfaces"
 maintainer="Oliver Kiddle <okiddle@yahoo.co.uk>"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/python-click/template]: BSD-3-Clause

Comment (xaltsc) [reply srcpkgs/python-click/template]: done

**Comment by maxice8** [comment]
Please check all packages that depend on `python-click' and `python3-click` to see if they work with click-7.x , some like gandi-cli only work with click-6.x

**Comment by xaltsc** [comment]
gandi-cli supports click7.0 on the lastest version (and only support the latest version). I'm going to update it.

**Comment by maxice8** [comment]
On Tue, Apr 09, 2019 at 10:22:42PM +0000, xaltsc wrote:
> gandi-cli supports click7.0 on the lastest version (and only support the latest version). I'm going to update it.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10624#issuecomment-481462356

grep for python-click and python3-click to see all the packages that need checking.

**Comment by xaltsc** [comment]
I did that and went through the setup.py on their repos, all seem compatible with click>=7.

**Comment by xaltsc** [comment]
gandi-cli builds on my PC (which uses glibc) but doesn't seem to build with travis on glibc for an unexplained reason.

**Comment by xaltsc** [comment]
done

**Comment by xaltsc** [comment]
Perhaps I should remove the latest changes (fix for the README) as soon as #10815 is resolved ?

**Comment by maxice8** [comment]
Please remove the fix, it is a problem in our implementation of travis-ci

**Comment by xaltsc** [comment]
Done. Should be ready to merge.

**Pull request merged**